sktime/benchmarking/tests/test_classification.py

```diff
@@ -144,3 +166,38 @@ def test_add_multiple_task(tmp_path):
         ),
         results_df["validation_id"],
     )
+
+
+@pytest.mark.skipif(
+    not run_test_module_changed("sktime.benchmarking"),
+    reason="run test only if benchmarking module has changed",
+)
+def test_multiple_dataset_format(tmp_path):
```

Comment:
make sure to decorate all tests that download data with `pytest.mark.datadownload`